### ISSUE TITLE
Starting Zettlr triggers installing Command Line Developer Tools

### ISSUE THREAD

**Issue by dimusfeldt** [Zettlr/Zettlr#5534]
### Description

On startup Zettlr issues a git command which tries to install the Command Line Developer Tools if they are not already installed. In turn the application “Install Command Line Developer Tools” from CoreServices is started and prompts the user to install the tools. 
The dialog reads “The 'git' command requires the Command Line Developer Tools. Install this software now? Choose “install” to load and install the Command Line Developer Tools.” (Or at least thats my translation from German.)

### Reproducing

Start Zettlr without Command Line Developer Tools installed.

### Zettlr version

3.4.1

### Installation Method

From the Website or GitHub

### Your Platform

- [ ] Windows
- [X] macOS
- [ ] Linux

### Architecture

- [X] Intel 64bit
- [ ] ARM 64bit (includes Apple Silicon)

### Operating System Version

Sequoia 15.2

**Comment by sensologica** [comment]
That sounds... strange. @nathanlesage, have you seen this?

**Comment by nathanlesage** [comment]
Yes, on Macs that don't have the command line tools installed, calling the git binary will trigger this dialog.

**Comment by benniekiss** [comment]
Closing since this is not really a bug.

**Comment by nathanlesage** [comment]
Also, this precise thing has indeed been fixed a few versions ago.
